test/e2e/support/gutenberg-commands.js

```diff
@@ -4,7 +4,7 @@ Cypress.Commands.add( 'newPost', () => {
 
 Cypress.Commands.add( 'visitAdmin', ( adminPath ) => {
 	cy.visit( '/wp-admin/' + adminPath ).location( 'pathname' ).then( ( path ) => {
-		if ( path === '/wp-login.php' ) {
+		if ( path.endsWith( '/wp-login.php' ) ) {
```

Comment:
I don't think we polyfill these ATM with Babel. I've been made aware of this several times by @aduth . :) Maybe use the loads version here instead?

Reply:
Though, this probably runs in the Cypress runner which doesn't have the same constraints as the browser (or all browsers)

Reply:
Yes, it's node 8 + electron or Chrome. Travis is happy so are we :)

Reply:
Oh, stupid me, I just saw the lines and my alarm bells went off without realising this is Cypress. :D